sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/CodeCoverageProvider.java

```diff
@@ -137,7 +137,7 @@ public List extensions() {
   public class UnitTestCoverageAggregator extends CoverageAggregator {
 
     public UnitTestCoverageAggregator(Configuration configuration, FileSystem fileSystem,
-                                      AnalysisWarnings analysisWarnings) {
```

Comment:
Also here, this change does not respect our convention.

Reply:
I think the IDE configuration needs to be changed to avoid this kind of changes in the future.

Reply:
Same as https://github.com/SonarSource/sonar-dotnet/pull/3531#discussion_r473833795

That's not something that can be fixed in this PR.